pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -546,12 +589,12 @@ func (l *listUsersQuery) expandExclusion(
 	ctx context.Context,
 	req *internalListUsersRequest,
 	rewrite *openfgav1.Userset_Difference,
-	foundUsersChan chan<- *openfgav1.User,
+	foundUsersChan chan<- foundUser,
 ) expandResponse {
 	ctx, span := tracer.Start(ctx, "expandExclusion")
 	defer span.End()
-	baseFoundUsersCh := make(chan *openfgav1.User, 1)
-	subtractFoundUsersCh := make(chan *openfgav1.User, 1)
+	baseFoundUsersCh := make(chan foundUser, 1)
+	subtractFoundUsersCh := make(chan foundUser, 1)
```

Comment:
FWIW during performance testing, having this channel as size 1 caused back pressure and modest performance degradation. Increasing the size to be much larger yielded a quick win with negation performance; it's really not great to start.

Reply:
Let's address the performance concerns and findings in a follow up related specifically to those findings.